ide/lsp.client/src/org/netbeans/modules/lsp/client/bindings/MarkOccurrences.java

```diff
@@ -101,6 +101,9 @@ private OffsetsBag computeHighlights(Document doc, int caretPos) {
         try {
             List<? extends DocumentHighlight> highlights =
                     server.getTextDocumentService().documentHighlight(new DocumentHighlightParams(new TextDocumentIdentifier(uri), Utils.createPosition(doc, caretPos))).get();
+            if(highlights == null) {
```

Comment:
Please format.

Reply:
Ah, you should create a branch when you create a PR. (e.g. `netbeans-5841`) See also: https://netbeans.apache.org/participate/submit-pr.html